src/components/CategoryEditTree.vue

```diff
@@ -70,8 +75,15 @@ export default {
         id: 0, // FIXME: Use ID assigned to category in vuex store, in order for saves to be uniquely targeted
         name: null,
         rule: {},
+        productivity: -1,
         parent: [],
       },
+      productivity_description:{
```

Comment:
I don't think this is a good way to do it.

Don't ever set "Unknown Productivity", in that case just don't set the productivity for that rule and make the transform skip that rule.

Then for example set 0 as neutral, 1 as productive and -1 as unproductive. Then you could for example later give the productivity score for a day/week/month/year a scale from -1 to 1

Reply:
@johan-bjareholt 
are you saying to completely remove the attribute if it's never set? If we don't set it to anything, then some categories will have a productive attribute, some won't, which might complicate the checking that needs to be done when we're parsing events. What do you think about flipping it instead to: "not productive":-1, "neutral":0, "productive":1, "unknown":2.

Reply:
>  What do you think about flipping it instead to: "not productive":-1, "neutral":0, "productive":1, "unknown":2.

See my comment below about I'd prefer if it was a range from -1 to 1 rather than static enums.